noobaa_sa/bucket.py

```diff
@@ -87,22 +90,65 @@ def delete(self, bucket_name, config_root=None):
         if config_root is None:
             config_root = self.config_root
         log.info(f"Deleting {bucket_name} Bucket from NSFS")
-        cmd = f"sudo /usr/local/noobaa-core/bin/node {self.manage_nsfs} bucket delete --name {bucket_name} --config_root {config_root}"
+        cmd = f"{self.base_cmd} bucket delete --name {bucket_name} --config_root {config_root}"
         retcode, stdout, stderr = self.conn.exec_cmd(cmd)
         if retcode != 0:
             raise e.BucketDeletionFailed(f"Deleting bucket failed with error {stderr}")
-        
         log.info(stdout)
         log.info("Bucket deleted successfully")
 
 
-    def update(self, bucket_name):
+    def update(self, bucket_name, config_root=None, **kwargs):
         """
         Bucket update
 
         Args:
             bucket_name (str): Bucket name to be updated
+            config_root (str): Path to config root
+        """
+        if config_root is None:
+            config_root = self.config_root
+        self.update_data = kwargs
+        update_cmd = ""         
+        if "new_name" in self.update_data:
+            update_cmd = f"--new_name {self.update_data.get('new_name')}"
+        #if "email" in self.update_data:
+        #    update_cmd = f"--email {self.update_data.get('email')}"
```

Comment:
If the plan is to implement the necessary support for updating email in a follow-up PR, please add a TODO comment above these commented-out lines.

Reply:
Fixed now